src/page-modules/contact/layouts/complaint-sub-page-layout.tsx

```diff
@@ -0,0 +1,68 @@
+import { PropsWithChildren } from 'react';
+import { SectionCard } from '../components/section-card';
+import { Input } from '../components/input';
+import { useRouter } from 'next/router';
+import { PageText } from '@atb/translations';
+
+export type ComplaintSubPageLayoutProps = PropsWithChildren<{
+  title: string;
+}>;
+
+function ComplaintSubPageLayout({ children }: ComplaintSubPageLayoutProps) {
```

Comment:
I think this could be `ComplaintPageLayout`

Reply:
Updatet now👍